io.github.Hexchat.Plugin.Perl.json

```diff
@@ -12,6 +12,10 @@
     "prepend-path": "/app/extensions/Perl/bin"
   },
   "modules": [
+    "shared-modules/dbus-glib/dbus-glib-0.110.json",
```

Comment:
Actually I think we can probably get rid of this entirely, just build hexchat with `-Dwith-dbus=false`